apps/docs/pages/guides/database/extensions/pg_net.mdx

```diff
@@ -145,7 +145,211 @@ request_id
 (1 row)
 ```
 
-After triggering `http_post`, use [`http_get_result`](#http_get_result) to get the result of the request.
+## `http_delete`
+
+Creates an HTTP DELETE request, returning the request's ID. HTTP requests are not started until the transaction is committed.
+
+### Signature [#post-signature]
+
+<Admonition type="caution">
+
+This is a Postgres [SECURITY DEFINER](https://supabase.com/docs/guides/database/postgres/row-level-security#use-security-definer-functions) function
+
+</Admonition>
+
+```sql
+net.http_delete(
+    -- url for the request
+    url text,
+    -- key/value pairs to be url encoded and appended to the `url`
+    params jsonb default '{}'::jsonb,
+    -- key/values to be included in request headers
+    headers jsonb default '{}'::jsonb,
+    -- the maximum number of milliseconds the request may take before being canceled
+    timeout_milliseconds int default 2000
+)
+    -- request_id reference
+    returns bigint
+
+    strict
+    volatile
+    parallel safe
+    language plpgsql
+    security definer
+```
+
+### Usage [#delete-usage]
+
+```sql
+select
+    net.http_delete(
+        'https://dummy.restapiexample.com/api/v1/delete/2'
+    ) as request_id;
+----------
+         1
+(1 row)
+```
+
+## Analyzing Responses
```

Comment:
```suggestion
## Analyzing responses
```

Our heading style is [sentence case](https://github.com/supabase/supabase/blob/master/apps/docs/CONTRIBUTING.md#styling-formatting-and-grammar)